cli/tests/integration_tests.rs

```diff
@@ -4994,6 +4994,21 @@ fn standalone_runtime_flags() {
     .contains("PermissionDenied: write access"));
 }
 
+#[test]
+fn denort_direct_use_error() {
+  let output = Command::new(util::denort_exe_path())
+    .current_dir(util::root_path())
+    .stderr(std::process::Stdio::piped())
+    .spawn()
+    .unwrap()
+    .wait_with_output()
+    .unwrap();
+  assert!(!output.status.success());
+  let stderr_str = String::from_utf8(output.stderr).unwrap();
+  assert!(util::strip_ansi_codes(&stderr_str)
+    .contains("This executable is used internally by 'deno compile', it is not meant to be invoked directly."));
```

Comment:
Maybe checking the exact error message isn't necessary. Having a check that it returns a non-zero error code is good tho.